velociraptor/autoplotter/lines.py

```diff
@@ -205,12 +214,12 @@ def create_line(
 
         if self.median:
             self.output = lines.binned_median_line(
-                x=masked_x, y=masked_y, x_bins=self.bins
-            )
+                x=masked_x, y=masked_y, x_bins=self.bins,
+                return_additional=self.return_additional)
         elif self.mean:
             self.output = lines.binned_mean_line(
-                x=masked_x, y=masked_y, x_bins=self.bins
-            )
+                x=masked_x, y=masked_y, x_bins=self.bins,
+                return_additional=self.return_additional)
         elif self.mass_function:
             self.output = create_mass_function_given_bins(
```

Comment:
Let's just force these to all have the same signature, so for the mass function you can expand this to be
```
self.output = *create_mass_function_given_bins(
   ....
), unyt_array([], units=masked_x.bins), unyt_array([], units=masked_y.bins)
```

Reply:
This is actually what I had in the first place. But then I saw that the function `create_line()` is also called in 
```
_make_plot_massfunction(...) 
_make_plot_scatter(..)
```
wherein it is assumed to output 3-length tuple (not 5-length). Thus, in order to avoid the error (expected 3 got 5), I decided to always output 5-tuple for mean & median and 3-tuple for mass_function & 2dhistorgram. What do you think?

Reply:
No, just change it there as well.

Reply:
To be more precise, the problem was the line
```
self.mass_function_line.create_line(
            x=x, y=None, box_volume=catalogue.units.comoving_box_volume
        )
```

Reply:
I see - I would much rather have the code be consistent than go for the fewest changes. So please also make the change wherever it's appropriate.

Reply:
Alright, I will implement these changes. I agree with all the other comments.